app/assets/stylesheets/components/_firm.scss

```diff
@@ -0,0 +1,44 @@
+.firm {
```

Comment:
Is this completely separate to `result`? I know they're displaying similar things. More a design question really, should we bring the results page more inline with this style?